lib/Unicode/UCD.t

```diff
@@ -834,6 +834,13 @@ is($ret_len, 5, "... and the returned length is 5");
 ok(! defined num("98765\N{FULLWIDTH DIGIT FOUR}", \$ret_len),
    'Verify num("98765\N{FULLWIDTH DIGIT FOUR}") isnt defined');
 is($ret_len, 5, "... but the returned length is 5");
+{
+    local $@;
+    my $ret_len = [ 5 ];
+    eval { num("98765", \$ret_len); };
+    like($@, qr/::num: second parameter must be a scalar reference/,
+        "num: Incorrect type for second parameter; must be scalar ref");
+}
```

Comment:
`\$ret_len` is a reference to a scalar (that happens to contain a reference), but this still fails.